### DISCUSSION TITLE
Support integrating tinyMCE with CMS to customize  the insert link function

### DISCUSSION THREAD

**Discussion by bshoemate** [tinymce/tinymce#8700] [General]
We are using TinyMCE as a part of a headless CMS solution and would like to customize the insert link feature to create an interface to browse and search the local content. Can someone point me in the right direction to get started? I'm looking for a developer I could hire to help with this.

**Comment by lostkeys** [comment]
Hi @bshoemate,

The general idea is that you create a custom button that opens your own link interface, [here you can see some examples](https://tinymce.github.io/tinymce-demos/index.html). Then use for example the [createlink command](https://www.tiny.cloud/docs/tinymce/6/editor-command-identifiers/) to insert the link into TinyMCE.

— Fredrik, Product Manager
